data/provider/src/main/java/tech/pegasys/teku/api/ExecutionClientDataProvider.java

```diff
@@ -18,7 +18,7 @@
 
 public class ExecutionClientDataProvider implements ExecutionClientEventsChannel {
 
-  private final AtomicBoolean isExecutionClientAvailable = new AtomicBoolean(false);
+  private final AtomicBoolean isExecutionClientAvailable = new AtomicBoolean(true);
```

Comment:
This was a silly mistake... After I updated the code on `Web3JClient` to avoid sending events for every successful request, I didn't notice that we would never go through the "recover" path and publish the event that EL is available. So if your communication with the EL never fails and recovers, it will never register as available.